tools/dependencies-report/src/main/java/org/logstash/dependencies/ReportGenerator.java

```diff
@@ -45,32 +45,33 @@ public boolean generateReport(
             readJavaDependenciesReport(stream, dependencies);
         }
 
-        Map<String, String> licenseMapping = new HashMap<>();
+        Map<String, String[]> licenseMapping = new HashMap<>();
         readLicenseMapping(licenseMappingStream, licenseMapping);
         List<String> acceptableLicenses = new ArrayList<>();
         readAcceptableLicenses(acceptableLicensesStream, acceptableLicenses);
         for (Dependency dependency : dependencies) {
             String nameAndVersion = dependency.name + ":" + dependency.version;
             if (licenseMapping.containsKey(nameAndVersion)) {
-                String mappedLicense = licenseMapping.get(nameAndVersion);
+                String[] mappedLicense = licenseMapping.get(nameAndVersion);
```

Comment:
Can we avoid `string[]` , while they are compact they make it hard to read the code. Can we make a value type instead? I know its more code, but I think it's worth it. We don't need to use accessors, just an object with public properties is fine.